src/utils/send.js

```diff
@@ -41,6 +73,28 @@ export async function createTokenAccountTransaction({
   };
 }
 
+export async function createTokenAccount({
+  connection,
+  wallet,
+  mintPublicKey,
+}) {
+  const { transaction, signer } = await createTokenAccountTransaction({
+    connection,
+    wallet,
+    mintPublicKey,
+  });
+
+  return await sendTransaction({
+    transaction,
+    signers: [signer, wallet.publicKey],
```

Comment:
`wallet.publicKey` needs to be specified before `signer` because whichever account is specified first is used to pay for transaction fees, otherwise the transaction fails because the newly-created account doesn't have any SOL to start with.